src/main/java/com/pingidentity/spring/example/config/OAuth2LoginSecurityConfig.java

```diff
@@ -26,11 +28,6 @@ protected void configure(HttpSecurity http) throws Exception {
         .antMatchers("/login", "/js/**", "/css/**", "/img/**", "/webjars/**").permitAll()
         .anyRequest().authenticated()
 
-        // Enable ssl
-        .and().requiresChannel()
-        .anyRequest()
-        .requiresSecure()
```

Comment:
Did you test that https is working without this part while having only
```server.ssl.key-alias=mycert
server.ssl.key-store-password=s$cret
server.ssl.key-store=classpath:zetcode.p12
server.ssl.key-store-provider=SUN
server.ssl.key-store-type=PKCS12``` ?

Reply:
Since HTTP is the default protocol for the samples  - we don't need this config anymore. Meanwhile, HTTPS is optional and configurable by every sample for its own.
I have tested config on the sample side it is working as expected.

Reply:
Pls add mentioning in REDAME about adding this code in case of switching to https

Reply:
Done!